api/tests/core/users/test_generator.py

```diff
@@ -157,3 +157,13 @@ def test_profile_completion_is_consistent_with_user_data(self):
         assert profile_completion.resultContent["firstName"] == user.firstName
         assert profile_completion.resultContent["lastName"] == user.lastName
         assert profile_completion.resultContent["postalCode"] == user.postalCode
+
+    def test_identity_check_is_consistent_with_user_data(self):
+        user_data = users_generator.GenerateUserData(
+            age=users_constants.ELIGIBILITY_AGE_18,
+            step=users_generator.GeneratedSubscriptionStep.IDENTITY_CHECK,
+        )
+        user = users_generator.generate_user(user_data)
+        identity_check = user.beneficiaryFraudChecks[-1]
+        assert identity_check.resultContent["first_name"] == user.firstName
+        assert identity_check.resultContent["last_name"] == user.lastName
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏Et `birth_date` ?